src/utilities/RecentFilesManager.java

```diff
@@ -0,0 +1,16 @@
+package utilities;
+
+import javax.swing.*;
+import java.awt.event.ActionListener;
+
+public class RecentFilesManager {
+    private JMenu recentFilesMenu;
+
+    public RecentFilesManager(JMenu recentFilesMenu) {
+        this.recentFilesMenu = recentFilesMenu;
+    }
+
+    public void AddRecentFile(String filePath, ActionListener actionListener) {
+        this.recentFilesMenu.add(MenuItemUtility.GetMenuItem(filePath, actionListener));
```

Comment:
This is a really nice way to add recent items to the menu. I tried using Preferences, which stores the recent items as a key/value pair, but it was a little difficult to deal with. Does this store recent items only for the session?